code/obj/machinery/plantpot.dm

```diff
@@ -138,6 +140,35 @@
 
 		frequency.post_signal(src, signal)
 
+	on_reagent_change()
```

Comment:
`process()` is already called like once every three seconds, so I dunno how necessary this proc and its contents are. Depends on how instant you care for this to be I guess.

Reply:
It's far more responsive to player action.